docs/manual/src/internals/foreign_calls.md

```diff
@@ -0,0 +1,84 @@
+# Rust -> Foreign calls
+
+How can Rust code make a call into the foreign language?
+The [lifting and lowering](./lifting_and_lowering.md) docs describe this at a high-level, this document will explore the details.
+
+This document only describes non-async calls.
+See the [Async FFI](./async-ffi.md) docs for a description of async calls.
+
+## VTables
+
+One major difference from a [Foreign -> Rust call](./rust_calls.md), is that the Rust code does not have access to a library with functions it can call.
```

Comment:
This only applies to objects and traits, not top-level functions, right? So maybe make that clear?

Reply:
yeah, sorry, I was confused :) We don't have plain foreign functions (yet? ;) - they are always in an interface. Maybe a sentence like "because foreign bindings are only called via an interface, the foreign references are all managed via a vtable" or similar - that was an unmentioned difference between the 2 docs I noticed.